.github/workflows/version-check.yml

```diff
@@ -39,4 +39,4 @@ jobs:
             exit 1
```

Comment:
Line above needs update:
"Any PR getting merged to master requires a version update"

Reply:
Good catch, will update the print statement.